lib/api/entries/index.js

```diff
@@ -96,8 +96,22 @@ function configure (app, wares, ctx) {
       // such as enforcing a type property to exist, are followed.
       return res.format({
         text: function ( ) {
+          res.set("Content-Type", 'text/plain');
           // sgvdata knows how to format sgv entries as text
-          es.pipeline(output, sgvdata.format( ), res);
+          es.pipeline(output, sgvdata.format( ), es.writeArray(function (err, out) {
+            res.send(out.join(""));
+          }));
+        },
+        csv: function ( ) {
+          // sgvdata knows how to format sgv entries as text
+          res.set("Content-Type", 'text/plain');
+          var csvpipe = require('sgvdata/lib/text')({ format: ',', parse: /[\t,]/ });
+          sgvdata.mapper(csvpipe.format), es.join("\n")
+          es.pipeline(output,
+            sgvdata.mapper(csvpipe.format), es.join("\n"),
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2165370020/issues/source?bid=2452379&fileBranchId=3160529#l111)